content/docs/news/release_announcements/keptn-100/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Keptn 1.0.0
+weight: 64
+---
+
+## [1.0.0](https://github.com/keptn/keptn/compare/0.19.0...1.0.0) (2022-12-12)
+
+---
+
+Keptn 1.0.0 is the first major release and the first LTS (Long Term Support) release of Keptn.
+Keptn LTS versions support is guaranteed for at least 6 months after the initial release. 
+The Keptn community announces the LTS end-of-life within 3 months in advance, using the community announcement channel [#announcements](https://app.slack.com/client/TFHDUSPB7/CFH4VK9GT) over Slack.
+For this, we introduced [scarf.sh](https://about.scarf.sh/) to track download data to decide whether the support of LTS versions will be extended.
+
+---
+
+### ⚠ BREAKING CHANGES
+
+* **installer:** We are introducing scarf.sh for download tracking to have valuable information that will help us decide whether support for Keptn LTS will be extended or not.
+* **shipyard-controller:** Keptn returns 404 instead of 400 HTTP error code when creation of the project fails when upstream repository is not found. Also returns 409 instead of 500 HTTP error code when creation or update of project fails due to upstream repository is already initialized.
```

Comment:
```suggestion
* **shipyard-controller:** Keptn returns 404 instead of 400 HTTP error code when creation of the project fails because the upstream repository is not found. Also returns 409 instead of 500 HTTP error code when creation or update of project fails because the upstream repository is already initialized.
```